apps/os/app/routes/org/estate/integrations/mcp-params.tsx

```diff
@@ -116,7 +116,10 @@ function MCPParams() {
     try {
       await saveParams({
         estateId,
-        connectionKey,
+        connectionId,
+        serverUrl,
+        mode: mode as "personal" | "company",
```

Comment:
```suggestion
        mode,
```

looks like it's defined as an enum in zod anyway